src/response.ts

```diff
@@ -1,6 +1,6 @@
 import Headers from './headers';
 
-export default interface Response<T = any> {
+export default interface Response<T = unknown> {
```

Comment:
I'm not sure if there's a need to provide `unknown` as the default value for the class variable. I think you can just omit the default value and leave it as `<T>`. That makes more sense to me than providing unknown as the default value, which you have to provide an actual value anyway.